content/5_Plat/RURQ.mdx

```diff
@@ -221,9 +221,171 @@ queries.
 	</Resource>
 </Resources>
 
-### Implementation
+### Solution - Range Updates & Sums
 
-<IncompleteSection />
+<Spoiler title="Solution">
+
+This question asks you to support the following types of queries. 
+
+- Add a value to all elements within the range $[a,b]$
+
+- Set all values within the range $[a,b]$ to a certain value 
+
+- Find the sum of all values in range $[a,b]$
+
+Consider the first two types of queries, a lazy tag will be created in each node of the tree for each type. In this solution, $lz1$ will represent the lazy tag for the range add query and $lz2$ will represent the lazy tag for the range set query. 
```

Comment:
would prefer more informative variable names (e.g. `lz_set, lz_add`) even though it makes the code longer

Reply:
I changed lz1 and lz2 to lzAdd and lzSet respectively.